cms/static/js/spec/views/course_video_settings_spec.js

```diff
@@ -128,16 +166,10 @@ define(
             });
 
             it('does not populate transcription plans if transcription plans are not provided', function() {
-                // First detroy old referance to the view.
-                destroyCourseVideoSettingsView();
-
                 // Create view with empty data.
-                renderCourseVideoSettingsView(null, null);
+                renderCourseVideoSettingsView();
 
-                expect($courseVideoSettingsEl.find('.transcript-provider-group').html()).toEqual('');
-                expect($courseVideoSettingsEl.find('#transcript-turnaround').html()).toEqual('');
-                expect($courseVideoSettingsEl.find('#transcript-fidelity').html()).toEqual('');
-                expect($courseVideoSettingsEl.find('#transcript-language').html()).toEqual('');
+                expect($courseVideoSettingsEl.find('.course-video-transcript-preferances-wrapper').html()).toEqual('');
```

Comment:
What will be rendered in this case? Is this the `N/A` case? If yes then we need to verify that.